app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragment.kt

```diff
@@ -43,7 +43,7 @@ import kotlin.math.min
 import kotlin.math.roundToInt
 
 class WeekFragment : Fragment(), WeeklyCalendar {
-    private val WEEKLY_EVENT_ID_LABEL = "event_id_label"
+    private val WEEKLY_EVENT_ID_LABEL = "eventIdLabel"
```

Comment:
guess this wasnt supposed to be edited

Reply:
yeah, I may have accidentally renamed a few things. I'll rename them before marking this for review...

Reply:
fixed that too